ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -28,6 +30,17 @@
 
   public static final String KSQL_SERVER_URL = "ksql.server.url";
 
+  public static final String KSQL_USERNAME = "ksql.username";
```

Comment:
nit: rename to clarify that this is used for basic auth
```suggestion
  public static final String KSQL_BASIC_AUTH_USERNAME = "ksql.auth.basic.username";
```
and similarly for password below.